yarn.lock

```diff
@@ -21119,7 +21119,7 @@ wrappy@1:
   version "1.0.2"
   resolved "https://registry.yarnpkg.com/wrappy/-/wrappy-1.0.2.tgz#b5243d8f3ec1aa35f1364605bc0d1036e30ab69f"
 
-write-file-atomic@2.4.1, write-file-atomic@^2.0.0, write-file-atomic@^2.3.0:
+write-file-atomic@2.4.1, write-file-atomic@^2.0.0, write-file-atomic@^2.3.0, write-file-atomic@^2.4.2, write-file-atomic@^3.0.0:
   version "2.4.1"
```

Comment:
Hmm, this part looks pretty weird - how `write-file-atomic@^2.4.2, write-file-atomic@^3.0.0` resolve to 2.4.1?

Reply:
It is weird indeed, this is the result of running `yarn workspace gatsby-remark-responsive-iframe remove bluebird`, which doesnt have any dependencies and therefore shouldnt update yarn.lock at all?

I think I should be able to restore yarn.lock without affecting anything

Reply:
So I previously added a new package to this plugin (in #18266), which moved write-file-atom 2.4.2 and 3.0.0 to their own entries in yarn.lock

This change here actually reverses that, restoring this line to what it was before the dependencies changes.

I'm... confused

Reply:
Don't worry about this too much - just something that caught my eye. I'll pull this locally and see if I get same behaviour with lock file

Reply:
Seems like we have yarn resolutions in our main package.json which would explain this - https://github.com/gatsbyjs/gatsby/blob/0ba9890f9e92b9eeb644e325825be263462064c6/package.json#L112-L114

But this was added ~2 months ago, so have no idea why lock didn't consolidate those before.

In any case - we should be good here

Reply:
Ah good to know, thank you. I've just looked at the previous PR, and the lock file changes was introduced randomly in a test snapshot update commit, so it's likely that I goofed something up.

I will pay more attention to unnecessary file changes in my commits.

Reply:
Don't sweat it - we are all good here! :)